ci/helm/values/values-nuxeo.yaml.gotmpl

```diff
@@ -19,10 +19,10 @@ service:
     dev.nuxeo.com/resource: service
 mongodb:
   enabled: true
-  host: mongodb.{{ .Namespace }}.svc.cluster.local
+  host: mongodb{{ eq .Environment.Name "functional-tests-2023" | ternary 6 4 }}.{{ .Namespace }}.svc.cluster.local
 elasticsearch:
   enabled: true
-  host: elasticsearch-master.{{ .Namespace }}.svc.cluster.local
+  host: {{ eq .Environment.Name "functional-tests-2023" | ternary "opensearch-cluster" "elasticsearch" }}-master.{{ .Namespace }}.svc.cluster.local
```

Comment:
For repository search purpose this would be better to have an if/else block on enabled condition.